spec/graphql/schema/scalar_spec.rb

```diff
@@ -143,8 +153,17 @@ class Query < GraphQL::Schema::Object
       }
       assert_equal [expected_error], result["errors"]
     end
-  end
 
+    it "makes a nice validation error for reuslt coercion" do
+      result = CoercionErrorSchema.execute("{ f2 }")
+      expected_error = {
+        "message" => "\"bad\" can't be Custom value",
+        "locations" => [{"line"=>1, "column"=>3}],
+        "path" => ["f2"],
```

Comment:
Interesting difference here. The input example above (from static validation) includes `query` in the path and this doesn't.

Looking at the spec, I don't think `query` should be included? But that's a separate issue (if at all).

Reply:
Yeah, `query` doesn't belong there. Related: https://github.com/rmosolgo/graphql-ruby/issues/4711